packages/legend-studio/src/stores/editor-state/element-editor-state/mapping/MappingTestState.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @finos/legend-studio/no-cross-workspace-source-usage */
```

Comment:
Let's remove this and fix the import by exporting the required type from its relevant package

Reply:
done